### PR TITLE
INGM-409: make new example about application using a pdo communication

### PR DESCRIPTION
**Pull request by WoodElv**
### Description

The application consists on setting a new position of a motor, and reading the actual position.

Fixes # INGM-409

### Type of change

- [x] Example of a simple application using PDOs

### Tests
- [x] The order of functions that are called in the example.

### Documentation

- [x] Update docstrings of every function, method or class that change.
- [x] Use [type hints](https://docs.python.org/3/library/typing.html) for every function and argument.
- [x] Build documentation locally to verify changes.

### Code formatting

- [x] Use black package to format the code: `black -l 100 ingeniamotion tests`. It is recommended to configure the code editor to automatically format the code using black with a max length line of 100.

### CODE REVIEW

File: examples/process_data_object.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    mc.capture.pdo.set_pdo_maps_to_slave(rpdo_map, tpdo_map)
+    # Start the PDO exchange
+    mc.capture.pdo.start_pdos()
+    time.sleep(waiting_time_for_pdo_exchange)
+    print(f"Actual Position: {actual_position.value}")
+    print(f"Position set-point: {position_set_point.value}")
+    # Increase the position set point
+    position_set_point.value += 500
+    time.sleep(waiting_time_for_pdo_exchange)
+    # Get the actual position value
```
Comment (mjedieckmann) [inline examples/process_data_object.py:32]: Maybe mention in the comment what value we expect to get here

File: examples/process_data_object.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import time
+
+from ingeniamotion.motion_controller import MotionController
+
+
+def update_position_value_using_pdo(mc: MotionController) -> None:
+    """Updates the position of a motor using PDOs.
+
+    Args:
+        mc : The controller where there are all functions to perform PDO exchange.
```
Comment (mjedieckmann) [inline examples/process_data_object.py:10]: "Controller with all the functions needed to perform a PDO exchange."

File: examples/process_data_object.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        "\\\\awe-srv-max-prd\\distext\\products\\CAP-NET\\firmware\\2.5.1\\cap-net-e_eoe_2.5.1.xdf"
+    )
+    mc.communication.connect_servo_ethercat_interface_index(
+        interface_index, slave_id, dictionary_path
+    )
+    print("Drive is connected.")
+    mc.motion.motor_enable()
+    print(f"Motor is enabled.")
+    update_position_value_using_pdo(mc)
+    print("Actual value is updated.")
```
Comment (mjedieckmann) [inline examples/process_data_object.py:56]: I think this print is not necessary

**Review by mjedieckmann — APPROVED** [review]

File: examples/process_data_object.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    # Stop the PDO exchange
+    mc.capture.pdo.stop_pdos()
+
+
+def main() -> None:
+    # Before running this example, the drive has to be configured for a your motor.
+    mc = MotionController()
+
+    # Modify these parameters to connect a drive
+    interface_index = 3
```
Comment (MartinALL91) [inline examples/process_data_object.py:44]: I think its best if we use the interface's IP

File: examples/process_data_object.py [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    # Create a TPDO map item
+    actual_position = mc.capture.pdo.create_pdo_item("CL_POS_FBK_VALUE")
+    # Create the RPDO and TPDO maps
+    rpdo_map, tpdo_map = mc.capture.pdo.create_pdo_maps([position_set_point], [actual_position])
+    # Map the PDO maps to the slave
+    mc.capture.pdo.set_pdo_maps_to_slave(rpdo_map, tpdo_map)
+    # Start the PDO exchange
+    mc.capture.pdo.start_pdos()
+    time.sleep(waiting_time_for_pdo_exchange)
+    print(f"Actual Position: {actual_position.value}")
```
Comment (MartinALL91) [inline examples/process_data_object.py:27]: You could use the PDO callbacks for this.

**Pull request merged**